spec/features/foods/foods_index_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+RSpec.describe 'foods index page', :vcr do 
```

Comment:
Great robust assertions and nice work using VCR! If this were a project I would be worried about this test being delicate (i.e. if the API adds a food, and VCR re-records, then total hits might go up to 49677 and break this test) but for an assessment this is a great level of detail